agent/bench-scripts/pbench-linpack

```diff
@@ -128,6 +128,7 @@ pushd /usr/local/linpack_11.1.3/benchmarks/linpack
 OMP_NUM_THREADS=$threads ./xlinpack_xeon64 < ./linpack.dat | tee $result_file
 pbench-stop-tools --group=$tool_group --iteration=$iteration --dir=$benchmark_results_dir
 pbench-postprocess-tools --group=$tool_group --iteration=$iteration --dir=$benchmark_results_dir
+pbench-metadata-log --group=$tool_group --dir=$benchmark_run_dir
```

Comment:
Add "`end`" argument, and invoke it with the "`beg`" argument between lines 123 and 124.